### PR TITLE
Added 3D STEP file support for kicad

### PR DESCRIPTION
**Pull request by groundstack**
With these changes I was able to compile kicad with support for 3D step model rendering.

Related to this issue:
https://github.com/void-linux/void-packages/issues/17818

I am new to void linux, first patch I submit, let's see if I am doing this properly :)

### CODE REVIEW

File: srcpkgs/kicad/template [hunk 1]
```diff
@@ -5,11 +5,11 @@ revision=1
 build_style=cmake
 configure_args="-DKICAD_BUILD_VERSION=${version} -DKICAD_SCRIPTING=ON
  -DKICAD_SCRIPTING_MODULES=ON -DKICAD_SCRIPTING_WXPYTHON=ON
- -DKICAD_SCRIPTING_ACTION_MENU=ON  -DBUILD_GITHUB_PLUGIN=ON -DKICAD_USE_OCE=OFF
+ -DKICAD_SCRIPTING_ACTION_MENU=ON  -DBUILD_GITHUB_PLUGIN=ON -DKICAD_USE_OCE=ON
  -DKICAD_SPICE=$(vopt_if spice ON OFF)"
 hostmakedepends="pkg-config swig wxWidgets-devel"
 makedepends="wxWidgets-devel wxPython-devel python-devel glew-devel cairo-devel
- libressl-devel boost-devel libcurl-devel glm occt-devel libgomp-devel
+ libressl-devel boost-devel libcurl-devel glm oce-devel libgomp-devel
```
Comment (Johnnynator) [inline srcpkgs/kicad/template]: Trailing space at the end of line
```suggestion
 libressl-devel boost-devel libcurl-devel glm oce-devel libgomp-devel
```

**Comment by groundstack** [comment]
My architecture is x86_64 musl.

Looking at the log from Travis CI it timed out around 90% of the compilation job. Seems same for glibc. What's next step??

**Comment by groundstack** [comment]
Who does the rev bump? I don't know how or if I can do it. 

Arch linux includes the OCE / Step functionality by default, as it is important feature to interface with mechanical engineers.

**Comment by ndowens** [comment]
You do, by editing the template file

**Comment by groundstack** [comment]
I've added revision number change. The bot tried to compile again, times out at 50 min where it is about 90%. How do we increase the compilation time?

**Comment by ndowens** [comment]
Can’t increase

**Comment by ericonr** [comment]
@groundstack the recommended OpenCascade library for KiCad is the OCC one (provided by the `occt` package). The reason KiCad wasn't built with STEP support was due to lacking the proper CMake config flag.

**Pull request closed without merging**